src/wp-includes/block-editor.php

```diff
@@ -513,6 +513,7 @@ function get_block_editor_settings( array $custom_settings, $block_editor_contex
 	}
 
 	$editor_settings['__unstableResolvedAssets']         = _wp_get_iframed_editor_assets();
+	$editor_settings['__unstableIsBlockBasedTheme']      = wp_is_block_theme();
```

Comment:
@ellatrix is this a new editor setting? If no, why "unstable"? Can if it stabilized (ie to avoid BC breaks later) for introduction into Core?

Reply:
I have no intention of stabilising it, it should just be for internal use. Currently it's just added for the purpose of iframing for block based theses, but in the future iframing should happen for all themes. So this setting will go away. Should I prefix it with `__internal`? Or what would you suggest?

Reply:
Thanks for sharing more context @ellatrix.

tl;dr
Its name is okay as is.

In this specific use case, this new setting's intent is for internal-only, not for use by extenders, and probably for removal in the future. Looking at the Gutenberg docs, `unstable` is the right naming convention to align to this setting's intent.